docker/rct/Dockerfile

```diff
@@ -0,0 +1,23 @@
+
+# RCT DOCKERFILE
+
+ARG BASE_IMAGE=exaworks/sdk-base
+FROM ${BASE_IMAGE}
+
+RUN mkdir -p /data/db /var/log/mongodb /var/run/mongodb
+ENV RADICAL_PILOT_DBURL=mongodb://localhost/am
+
+# RCT installation (use EnTK devel package temporary)
+RUN pip install \
+    radical.pilot \
+    git+https://github.com/radical-cybertools/radical.entk.git@devel
+
+# clone RP and RE to get the test suite
+RUN git clone -b master --single-branch https://github.com/radical-cybertools/radical.pilot.git \
```

Comment:
If we're just cloning the repo for the test suite, is there any reason why we wouldn't want to clone it every time we run the test suite? This way we could run against the most up to date test suite. A failing test could indicate that we need to update the container image

Reply:
I would think it is good to have repo (with examples) in sync with the installed version (kinda tests confirm that installed version is functional)

Reply:
That makes sense to me, great work!